src/Chain.php

```diff
@@ -337,6 +337,26 @@ public function validate(MessageStack $messageStack, Container $input, Container
     }
 
     /**
+     * Get all rules applied to this chain
+     */
+    public function getRules()
+    {
+        $rules = [];
+
+        /** @var Rule $rule */
+        foreach ($this->rules as $rule) {
+            $reflectionRule = new \ReflectionClass($rule);
+            $rules[] = [
+                'name' => $reflectionRule->getShortName(),
```

Comment:
`substr(get_class($rule), strrpos(get_class($rule), NAMESPACE_SEPARATOR));` that'd yield the same result, I think.